### DISCUSSION TITLE
Meshoptimizer library

### DISCUSSION THREAD

**Discussion by Pheng-AN** [zeux/meshoptimizer#599] [General]
I currently building AR Geospatial in Unity version 2022 on IOS platform after i added meshoptimizer library to my project and i counter the error it said Use of undeclared identifier '__builtin_wasm_memory_size' in my xcode version 14 this my file path:
/Users/Desktop/PRO-AR/Libraries/Plugins/x86_64/meshoptimizer/tools/wasmstubs.cpp
![Uploading Screenshot 2023-08-27 at 11.10.12 PM.png…]()

**Comment by zeux** [comment]
This file isn’t really part of the library - are you adding cpp files to the project yourself? In this case you just need files in src/ folder.

**Reply by Pheng-AN** [reply]
yes

**Reply by Pheng-AN** [reply]
could you give me tutorial for installing this meshoptimizer to my xcode
